flow/designs/asap7/aes/config.mk

```diff
@@ -7,8 +7,8 @@ export DESIGN_NAME            = aes_cipher_top
 export DESIGN_NICKNAME        = aes
 export DESIGN                 = aes
 
-export VERILOG_FILES          = $(sort $(wildcard $(abspath $(DESIGN_DIR)/../../src/$(DESIGN))/*.v))
-export SDC_FILE               = $(DESIGN_DIR)/constraint.sdc
+export VERILOG_FILES          = $(sort $(wildcard ./designs/src/$(DESIGN_NICKNAME)/*.v))
+export SDC_FILE               = ./designs/$(PLATFORM)/$(DESIGN_NICKNAME)/constraint.sdc
```

Comment:
Why change here and on other instances? I think using `DESIGN_DIR` is cleaner on the `export SDC_FILE`.

Reply:
Hi Vitor, it is because genMassive.script creates flow variant config mkfiles under the designs/{platform}/{design}/chunks/chunk0/ path. So the mk file refers the original mk file as "include $(realpath $(dir $(DESIGN_CONFIG))../../)/config.mk". And my runscript includes flow variant config mkfile, so $(DESIGN_CONFIG) variable is messed up.

Reply:
I am not sure I understand the issue. Nevertheless, this looks more like a patch and not a solution for the problem of "$(DESIGN_CONFIG) variable is messed up." Won't other things that depend on this variable also be broken? If this is the best we can do, we should document this somewhere at least -- when to use and when not to use.

Reply:
I will roll back this part until I find another solution.